### DISCUSSION TITLE
关于文本识别相似度不高导致经常停顿的问题

### DISCUSSION THREAD

**Discussion by viogami** [NateScarlet/auto-derby#354] [Q&A] [answered]
自动育成的时候，遇到识别文本相似度不高的时候，每次训练会重复识别多次，浪费很多时间。

只是育成部分马娘的时候会出现这个问题较多（比如阿尔丹，养内恰的时候就出现几次，人工确定就ok了）

请问出现这种情况是什么原因呢，我看到启动脚本设置了游戏界面大小，但是分辨率还是不足吗？
或者说怎么让70%多的相似度就可以继续运行呢
![(C84`74(PI8WMUA`Q@T%N~1](https://user-images.githubusercontent.com/90465552/186835667-5388faab-8c7b-409e-9028-a86eef9b827e.png)

![83C23AUZ{$EDPB@FME~B(48](https://user-images.githubusercontent.com/90465552/186835671-ec562b4a-6f51-4a05-8ea1-e659c92cd5aa.png)

**Comment by NateScarlet** [comment, marked as answer]
这个是样本不够导致的 项目自带的样本仅从测试截图里取

可以尝试用我的数据 [ocr_labels.csv](https://github.com/NateScarlet/auto-derby/files/9430559/ocr_labels.csv)

低相似度不提示可以用 no_ocr_prompt 插件，不过出错别找我
